app/components/markdown/markdown_table_image/markdown_table_image.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {PropTypes} from 'prop-types';
+import React from 'react';
+import {intlShape} from 'react-intl';
+import {Text} from 'react-native';
+
+import CustomPropTypes from 'app/constants/custom_prop_types';
+import {wrapWithPreventDoubleTap} from 'app/utils/tap';
+
+export default class MarkdownTableImage extends React.PureComponent {
+    static propTypes = {
+        children: PropTypes.node.isRequired,
+        source: PropTypes.string.isRequired,
+        textStyle: CustomPropTypes.Style.isRequired,
+        navigator: PropTypes.object.isRequired,
+        serverURL: PropTypes.string.isRequired,
+        theme: PropTypes.object.isRequired
+    };
+
+    static contextTypes = {
+        intl: intlShape.isRequired
+    };
+
+    handlePress = wrapWithPreventDoubleTap(() => {
+        const {navigator, theme} = this.props;
+
+        navigator.push({
+            screen: 'TableImage',
+            title: this.context.intl.formatMessage({
+                id: 'mobile.routes.tableImage',
+                defaultMessage: 'Image'
+            }),
+            animated: true,
+            backButtonTitle: '',
+            passProps: {
+                imageSource: this.getImageSource()
+            },
+            navigatorStyle: {
+                navBarTextColor: theme.sidebarHeaderTextColor,
+                navBarBackgroundColor: theme.sidebarHeaderBg,
+                navBarButtonColor: theme.sidebarHeaderTextColor,
+                screenBackgroundColor: theme.centerChannelBg
+            }
+        });
+    });
+
+    getImageSource = () => {
+        let source = this.props.source;
+
+        if (source.startsWith('/')) {
+            source = this.props.serverURL + '/' + source;
```

Comment:
can we use here `${this.props.serverURL}/${source}`, If I remember correctly we are using the syntax everywhere